Goals/Goals/DetailController.swift

```diff
@@ -31,12 +31,9 @@ class DetailController: UIViewController {
 
     titleTextField.isHidden = true
   }
-}
 
 // TODO: Add support to edit the other properties of a goal.
 
-extension DetailController {
```

Comment:
You are right. I actually inserted the definition in IB before adding the extension, exactly because of this problem. I just thought, they might fix it in an Xcode update and I do structurally prefer separating functionality into multiple extensions.